notify/email/email.go

```diff
@@ -239,6 +243,11 @@ func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 	multipartBuffer := &bytes.Buffer{}
 	multipartWriter := multipart.NewWriter(multipartBuffer)
 
+	id, err := generateMessageID()
```

Comment:
For sanity, you need to make sure that the `Message-Id` header doesn't already exist.

Reply:
done